spec/services/gpo_confirmation_exporter_spec.rb

```diff
@@ -17,6 +17,7 @@
           otp: 'ZYX987',
           issuer: issuer,
         },
+        created_at: '2018-06-29 01`:02:03'.to_datetime,
```

Comment:
I think the backtick in there is a typo? but it worked anyways
```suggestion
        created_at: '2018-06-29 01:02:03'.to_datetime,
```

Also I don't think that `String#to_datetime` is the most common way we build time objects? (I feel like `Time.zone.parse('...')` is more common but it's prob fine here

Reply:
Yeah, the specs seem split slightly in favor of `Time.zone.parse`. I'm gonna switch to that because it's a little clearer about the timezone in question.